configure.ac

```diff
@@ -4236,7 +4236,7 @@ case "$host" in
 		CPPFLAGS="$CPPFLAGS -D__ARM_EABI__"
 		;;
 	aarch64-*darwin*ilp32)
-		TARGET=ARM64;
+		TARGET=ARM6432
```

Comment:
Note that https://github.com/mono/mono/blob/972f29179dc11558902d89b55ae2f333c71a9aa9/configure.ac#L4442-L4445 already exists on master